coalib/__init__.py

```diff
@@ -6,6 +6,10 @@
 
 
 def get_version():
+    '''The coalib package is a collection of various subpackages
```

Comment:
Double quotes will essentially lead to a print statement which again will execute even in other processes such as using a bear to test a script. Therefore, I used a docstring. Added a newline before and after the quotes.